front/style/pages/_join.scss

```diff
@@ -0,0 +1,150 @@
+.join {
+    display: flex;
+    justify-content: space-between;
+    height: 100vh;
+    overflow: hidden;
+
+    @media (max-width: $tablet) {
+        flex-direction: column;
+        height: auto;
+    }
+
+    &__desc {
+        width: 35%;
+        height: 100vh;
+        transition: all 0.5s;
+
+        @media (max-width: $tablet) {
+            height: auto;
+            width: 100%;
+        }
+
+        h1 {
+            position: absolute;
+            margin: 10px;
+            z-index: 100;
+        }
+
+        .desc__texts {
+            position: relative;
+            @include vertical--align;
+
+            @media (max-width: $tablet) {
+                top: 0;
+                transform: translateY(0);
+                margin: 0 auto;
+                width: 100%;
+            }
+
+            span {
+                @include text--large;
+                @include text--bold;
+                font-family: 'Gillsans';
+                border-bottom: 10px solid $yellow;
+                box-shadow: inset 0 -5px 0 $yellow;
+                display: inline-block;
+                padding: 0 10px;
+                height: 30px;
+
+                @media (max-width: $tablet) {
+                    height: 25px;
+                }
+            }
+
+            p {
+                text-align: justify;
+                padding: 2em;
+            }
+        }
+    }
+
+    &__form {
+        width: 65%;
+        background-color: $gray--white;
+        height: auto;
+        overflow: scroll;
+        padding: 10% 0;
+
+        @media (max-width: $tablet) {
+            width: 100%;
+            overflow: hidden;
+            padding: 5% 0;
+        }
+
+        textarea:focus,
+        input:focus,
+        select:focus {
+            outline: 0;
+            border: 1px solid $green--other;
+            transition: all 1s;
+        }
+
+        input,
+        select {
+            padding-left: 12px;
+            padding-right: 12px;
+            border-radius: 3px;
+            border: solid 1px $gray--light;
+            height: 60px;
+            box-sizing: border-box;
+            font-size: 14px;
+            font-family: "Roboto", Helvetica, sans-serif;
+        }
+
+        select#user_registration_birthdate_day,
+        select#become_adherent_birthdate_day,
+        select#user_registration_birthdate_month,
+        select#become_adherent_birthdate_month {
+            width: 90%;
+            margin-right: 10px;
+        }
+
+        select#user_registration_birthdate_year,
+        select#become_adherent_birthdate_year {
+            width: 90%;
+        }
+
+        .form__captcha {
+            width: 100%;
+        }
+
+        .register__form--full {
+            width: 100%;
+        }
+
+        .form__checkbox input[type="checkbox"]:checked+label:before {
+            transition: all 1s;
+            border-color: $green--other;
+            background: $green--other;
+        }
+
+        .join__legal {
```

Comment:
is the `join` part necessary ? since we're already in `join`

Reply:
✅